proto/greenfield/storage/query.proto

```diff
@@ -148,6 +148,11 @@ service Query {
     option (google.api.http).get = "/greenfield/storage/groups_exist_by_id/{group_ids}";
   }
 
+  // Queries resource tags.
+  rpc QueryResourceTag(QueryResourceTagRequest) returns (QueryResourceTagResponse) {
```

Comment:
I think we dont have a new API.  The tags can be queried when query the resources, it should be one field of these resources I guess.

Reply:
Done